environment/requirements-win.txt

```diff
@@ -0,0 +1,29 @@
+notebook
+ipython>=6
+nbformat>=4
+pre-commit==2.20.0
+nbconvert>=5
+cython==0.29.32
+cmake==3.24.1
+nbqa==1.3.1
+pylint==2.14.3
+scour==0.38.2
+qiskit==0.42.1
+qiskit-ibm-runtime==0.9.3
+qiskit-nature==0.6.0
+qiskit-machine-learning==0.4.0
+qiskit-finance==0.3.2
+scikit-learn==1.0.1
+tensorflow==2.9.1
+matplotlib==3.5.2
+#pyscf==2.0.1
+--prefer-binary pyscf
```

Comment:
Similarly, instead of two requirements files, I think you should be able to do something like the following to change requirements based on system.

```
pyscf==2.0.1; sys_platform != 'win32'
--prefer-binary pyscf==2.0.1; sys_platform == 'win32'
```

Reply:
@frankharkins I will fix it by tomorrow night. Thanks so much for the comments